scripts/local_testnet/README.md

```diff
@@ -5,55 +5,99 @@ This setup can be useful for testing and development.
 
 ## Requirements
 
-The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH`.
+The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH` (run `echo $PATH` to view all `PATH` directories).
 
 
 MacOS users need to install GNU `sed` and GNU `grep`, and add them both to `PATH` as well.
 
-From the
-root of this repository, run:
+The first step is to install Rust and dependencies. Refer to the [Lighthouse Book](https://lighthouse-book.sigmaprime.io/installation-source.html#dependencies) for installation. We will also need [jq](https://jqlang.github.io/jq/), which can be installed with `sudo apt install jq`.
+
+Then, we clone the Lighthouse repository:
+```bash
+cd ~
+https://github.com/sigp/lighthouse.git
+cd lighthouse
+```
+We are now ready to build Lighthouse. Run the command:
 
 ```bash
 make
 make install-lcli
 ```
 
+This will build `lighthouse` and `lcli`. For `geth` and `bootnode`, go to [geth website](https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz) and download the `Geth & Tools`. For example, to download and extract `Geth & Tools 1.13.1`:
```

Comment:
Perhaps link this one instead which has links to macos and windows downloads?
https://geth.ethereum.org/downloads

Reply:
Fixed, thanks!